Lib/fontbakery/checkrunner.py

```diff
@@ -1234,8 +1238,17 @@ def _section_execution_order(self, section, iterargs
         continue
       full_order.append(item)
 
-    checks = [check for check in section.checks \
-                        if not explicit_checks or check.id in explicit_checks]
+    if explicit_checks:
+      checks = [
+          check for check in section.checks if check.id in explicit_checks
+      ]
+    elif exclude_checks:
+      checks = [
+          check for check in section.checks if check.id not in exclude_checks
+      ]
+    else:
+      checks = [check for check in section.checks]
```

Comment:
Isn't this the same as `checks = section.checks` ?

Also there should be code verifying whether the user passed both `explicit_checks` and `exclude_checks` because that should result in an error explaining that the options are mutually exclusive.